backend/Events/VoteEvents.js

```diff
@@ -15,41 +15,36 @@ function voteDay(io, socket, mafiaGame) {
         const room = mafiaGame.gameRoomsDict[socket.player.roomID];
         const voter = socket.player;
         const votee = voteForDTO.voteFor;
-        room.voteMapping[voter.nickname] = votee;
-        io.in(socket.player.roomID).emit(
-            'day-vote-update',
-            new ListVoteDTO(room.voteMapping)
-        );
+        room.voteHandler.daytimeVoteMap[voter.nickname] = room.getPlayerByNickname(votee).nickname;
```

Comment:
on subsequent day-votes, does the daytimeVoteMap get reset? does it matter?

Reply:
yeap resetted after every gamestate end (day, night and trial)